packages/cli/src/args.ts

```diff
@@ -0,0 +1,81 @@
+import chalk from 'chalk';
+import logo from './util/output/logo';
+import { getPkgName } from './util/pkg-name';
+
+export const help = () => `
+  ${chalk.bold(`${logo} ${getPkgName()}`)} [options] <command | path>
+
+  ${chalk.dim('Commands:')}
+
+    ${chalk.dim('Basic')}
+
+      deploy               [path]      Performs a deployment ${chalk.bold(
+        '(default)'
+      )}
+      dev                              Start a local development server
+      env                              Manages the Environment Variables for your current Project
+      git                              Manage Git provider repository for your current Project
+      help                 [cmd]       Displays complete help for [cmd]
+      init                 [example]   Initialize an example project
+      inspect              [id]        Displays information related to a deployment
+      link                 [path]      Link local directory to a Vercel Project
+      ls | list            [app]       Lists deployments
+      login                [email]     Logs into your account or creates a new one
+      logout                           Logs out of your account
+      pull                 [path]      Pull your Project Settings from the cloud
+      rollback             [url|id]    Quickly revert back to a previous deployment [beta]
+      switch               [scope]     Switches between teams and your personal account
+
+    ${chalk.dim('Advanced')}
+
+      alias                [cmd]       Manages your domain aliases
+      bisect                           Use binary search to find the deployment that introduced a bug
+      certs                [cmd]       Manages your SSL certificates
+      dns                  [name]      Manages your DNS records
+      domains              [name]      Manages your domain names
+      logs                 [url]       Displays the logs for a deployment
+      projects                         Manages your Projects
+      rm | remove          [id]        Removes a deployment
+      secrets              [name]      Manages your global Secrets, for use in Environment Variables
+      teams                            Manages your teams
+      whoami                           Shows the username of the currently logged in user
+
+  ${chalk.dim('Options:')}
+
+    -h, --help                     Output usage information
+    -v, --version                  Output the version number
+    --cwd                          Current working directory
+    -A ${chalk.bold.underline('FILE')}, --local-config=${chalk.bold.underline(
+  'FILE'
+)}   Path to the local ${'`vercel.json`'} file
+    -Q ${chalk.bold.underline('DIR')}, --global-config=${chalk.bold.underline(
+  'DIR'
+)}    Path to the global ${'`.vercel`'} directory
+    -d, --debug                    Debug mode [off]
+    --no-color                     No color mode [off]
+    -f, --force                    Force a new deployment even if nothing has changed
```

Comment:
Should `--force` be removed from the global options? It looks like it's only used in the `deploy`, `domains`, and `init` commands.

Reply:
IMO yes.

Reply:
Oh, it's not even a common arg now. Yeah, let's remove it. [`c2e0ca5` (#9912)](https://github.com/vercel/vercel/pull/9912/commits/c2e0ca562b1a3d3b877508bd3f91872bf5e99668)

Reply:
Removed from docs, too: [vercel/front@`95b71be` (#22110)](https://github.com/vercel/front/pull/22110/commits/95b71be4e41b901f7860e5128c73416ac493e963)